### PR TITLE
[SCHEDULED for Dec 1 do not merge before] feat(blog): conferences

### PR DESCRIPTION
**Pull request by janl**
Part 3 in our little series.

### CODE REVIEW

File: _posts/2015-12-1-conferences.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: "JSConf EU, CSSconf EU, Reject.js & OTSConf"
+author: Jan
+layout: blog-post-tgif
+comments: true
+---
+
+This is Part 2 of of our [little series about what has happened in Hoodie-land](http://hood.ie/blog/we-have-been-busy.html "We. Have. Been. Busy!") in the past <strike>three</strike>five months.
```
Comment (garrensmith) [inline _posts/2015-12-1-conferences.md]: Part 3?

**Comment by varjmes** [comment]
Group picture by myself -- 
varapproves

File: _posts/2015-12-1-conferences.md (hunk 1 above)
Comment (varjmes) [inline _posts/2015-12-1-conferences.md]: Part 2 should be Part 3. And "of of" should be "of" :)

File: _posts/2015-12-1-conferences.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+In order, we worked on OTSConf, Reject.js, CSSconf EU and JSConf EU and we got to meet **a lot** of great people! Here are just a few:
+
+<a data-flickr-embed="true"  href="https://www.flickr.com/photos/blank22763/21840012061/in/dateposted/" title="Untitled"><img src="https://farm1.staticflickr.com/584/21840012061_84207361b8_h.jpg" width="1600" height="1133" alt="Untitled"></a><script async src="//embedr.flickr.com/assets/client-code.js" charset="utf-8"></script>
+
+Featuring (from left to right): 
+[@janl](https://twitter.com/janl), [@boennemann](https://twitter.com/boennemann), [@misprintedtype](https://twitter.com/misprintedtype), [@christophwitzko](https://twitter.com/christophwitzko), [@lewiscowper](https://twitter.com/lewiscowper), [@hipsterbrown](https://twitter.com/hipsterbrown), [@anderspree](https://twitter.com/anderspree), [@espylaub](https://twitter.com/espylaub), [@gr2m](https://twitter.com/gr2m) & [@autofocus](https://twitter.com/autofocus).
+
+And to top things off, four Hoodie community members gave talks:
+
+- [Nick Hehr](https://twitter.com/hipsterbrown): [The other Side of Empathy](https://www.youtube.com/watch?v=6evp7YLDJrg)
```
Comment (varjmes) [inline _posts/2015-12-1-conferences.md]: "other" should probably be capitalised in Nicks' talk for consistency :)

**Comment by gr2m** [comment]
Love it :)

**Comment by janl** [comment]
We are live, thanks’s y’all for your comments! :)

**Pull request merged**